src/app/shared/pipes/duration.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import * as moment from 'moment';
+
+@Pipe({name: 'duration'})
+export class DurationPipe implements PipeTransform {
+  transform(minutes: number): string {
+    const duration = moment.duration(minutes, 'minutes');
+
+    return minutes >= 60
```

Comment:
please create separate constants for magic numbers

Reply:
I removed magic nubers for now. I'll create the constants-file in future if it will be needed